ci/kind/test-e2e-kind.sh

```diff
@@ -60,7 +60,7 @@ function quit {
     exit $result
   fi
   echoerr "Cleaning testbed"
```

Comment:
Same question, it's better to add this change to a separate PR.

Reply:
Changed back.